app/js/helper_profile_controller.js

```diff
@@ -33,6 +33,7 @@
             var updated = results[i].updated;
             results[i].updated_day = updated.split('T')[0];
             results[i].updated = Utils.time_since(new Date(updated));
+            results[i].human_meta = Utils.meta_to_human(results[i].metadata);
```

Comment:
nit: similar to the function i think it's better to have this be something like `results[i].meta_parsed`